src/app/components/Feed/Feed.tsx

```diff
@@ -48,15 +49,17 @@ function Feed({ username, onMatchClick }: FeedProps): JSX.Element {
       >
         Back to top
       </button>
-      {results.map((match) => (
-        <MatchItem
-          key={match._id}
-          match={match}
-          onClick={() => onMatchClick(match._id)}
+      {results.map((highlight) => (
+        <Highlight
+          key={highlight._id}
+          highlight={highlight}
+          onHighlightClick={onHighlightClick}
+          matchIsActive={true}
+          layout="full"
         />
       ))}
-      {matches?.results.length === 0 && (
-        <p className={classes.noMatches}>No matches found</p>
+      {highlights?.results.length === 0 && (
```

Comment:
just a minor improvement without any value
```suggestion
      {!highlights?.results.length && (
```